src/fights.ts

```diff
@@ -846,13 +846,30 @@ const freeFightSources = [
   ),
 
   new FreeFight(
-    () => have($item`[glitch season reward name]`) && !get("_glitchMonsterFights"),
-    () => {
-      restoreHp(myMaxhp());
-      retrieveItem($item`[glitch season reward name]`);
-      visitUrl("inv_eat.php?pwd&whichitem=10207");
-      runCombat();
-    }
+    () =>
+      have($item`[glitch season reward name]`) &&
+      !get("_glitchMonsterFights") &&
+      get("garbo_fightGlitch", false),
+    () =>
+      withMacro(
+        Macro.skill($skill`Curse of Marinara`)
+          .skill($skill`Conspiratorial Whispers`)
+          .skill($skill`Shadow Noodles`)
+          .item([$item`gas can`, $item`gas can`])
+          .externalIf(
+            get("lovebugsUnlocked"),
+            Macro.trySkill($skill`Summon Love Gnats`).trySkill($skill`Summon Love Mosquito`)
+          )
+          .trySkill($skill`Micrometeorite`)
+          .kill(),
+        () => {
+          restoreHp(myMaxhp());
+          retrieveItem($item`[glitch season reward name]`);
+          retrieveItem($item`gas can`, 2);
+          visitUrl("inv_eat.php?pwd&whichitem=10207");
+          runCombat();
+        }
+      )
```

Comment:
I think the biggest issue atm is just the stasis rounds that basicCombat does, but at least for me when I just added that max hp restore it basically stopped having any problems because I had enough hp to get through those rounds. Using gas cans is probably overkill.

Maximizing something like mainstat or Spell Damage percent might be effective as well.

Reply:
How many implements do you have? Could we add spicy tech for high implementations, or is that not really an issue?

Reply:
I'm at 490 implements and have two of them, so I'm dealing with a ~5k monster atm.

Reply:
I think I'd like to leave in the gas cans for high implementation levels--does 1000 seem reasonable? that's what I have as my catch. Remember, this needs to work on a broad array of accounts, but also because it's a user property people do know they're signing up for a chance of death